src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo_data.c

```diff
@@ -0,0 +1,248 @@
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 29, { { 0, 0, 0 }, 30 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 30, { { 0, 0, 0 }, 30 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 31, { { 0, 0, 0 }, 30 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 32, { { 0, 0, 0 }, 50 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 38, { { 0, 0, 0 }, 30 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 39, { { 0, 0, 0 }, 30 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 40, { { 0, 0, 0 }, 40 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 45, { { 0, 0, 0 }, 30 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 46, { { 0, 0, 0 }, 30 }, 100 },
+    },
+    {
+        {
+            ELEMTYPE_UNK2,
+            { 0xFFCFFFFF, 0x00, 0x04 },
+            { 0xFFCFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 47, { { 0, 0, 0 }, 40 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    {
+        COLTYPE_HIT3,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_PLAYER,
+        OC2_TYPE_1,
+        COLSHAPE_JNTSPH,
+    },
+    19,
+    sJntSphElementsInit,
+};
+
+Vec3f unkZeroVec = { 0.0f, 0.0f, 0.0f };
+
+u64 gBossDodongoLavaFloorLavaTex[] = {
```

Comment:
These variables are local to the file so I think they should be named as such
```suggestion
u64 sLavaFloorLavaTex[] = {
```